src/invoker/invoker.hpp

```diff
@@ -37,7 +37,7 @@ namespace big
 		constexpr void end_call()
 		{
 			// TODO: try to get rid of this
-			[[assume((cache_handlers(), m_are_handlers_cached == true))]];
+			[[assume((cache_handlers(), m_are_handlers_cached == true))]]

```

Comment:
These semi-colons don't belong affixed to an attribute anyway. To the compiler, you basically just said:

"Hello, compiler. Please assume that Cache Handler's member are handlers cached is true. Thank you.
(New command)
End of Line."

 Good find.